### ISSUE TITLE
Investigate and fix the ignored tests and evaluate whether some of them might need to be removed

### ISSUE THREAD

**Issue by carlspring** [strongbox/strongbox#1448]
# Task Description

As seen here:

![image](https://user-images.githubusercontent.com/1436265/64030618-d7c60300-cb3e-11e9-8296-350f597873ab.png)

There are currently several test cases which are ignored and need to either be investigated and fixed, or re-evaluated and possibly be removed.

# Tasks

The following tasks will need to be carried out:
* [ ] Fix those tests that make sense to be re-enabled.
* [ ] Drop obsolete tests.

# Useful Links

* [Strongbox: Writing Tests](https://strongbox.github.io/developer-guide/writing-tests.html)
* [Strongbox: JUnit User Guide](https://strongbox.github.io/developer-guide/junit-user-guide.html)
* [Strongbox: Writing Web Form Tests](https://strongbox.github.io/developer-guide/writing-web-form-tests.html)

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

**Comment by PedroHReis22** [comment]
can I take this one?

**Comment by carlspring** [comment]
Sure! Go ahead!

Feel free to join our [chat channel](https://chat.carlspring.org/).

**Comment by PedroHReis22** [comment]
Here is the outcome of what I found:
- `AqlParserTest`: one test is being skipped (`testValidQueryWithUpperLowercaseCheck`). The test case seems to be invalid as currently, AQL is case sensitive. Can I change the test adjusting the expectation?
- `LoggingManagementControllerTestIT`: all 16 tests are being skipped. On top of the class, there's a comment regarding migrating to spring boot actuator after #1000, however, as far as I saw, actuator logs endpoint weren't enabled in #1000. Is there any ticket tracking this need (if that is a real need)? Also, if enabled, logs are not failing.
- `ProxyRepositoryConnectionPoolConfigurationServiceImplIntegrationTest`: Tests are disabled but all the 6 tests are passing.
- `MavenProxyRepositoryProviderTestIT`: One test being skipped stating that needs to Docker needs to be in place. Is the docker setup done?
- `ChecksumCacheManagerTest`: 2 tests disabled but are passing

 May I have your thoughts regarding each of the cases?

**Comment by carlspring** [comment]
> Here is the outcome of what I found:
> 
> * `AqlParserTest`: one test is being skipped (`testValidQueryWithUpperLowercaseCheck`). The test case seems to be invalid as currently, AQL is case sensitive. Can I change the test adjusting the expectation?

The AQL is currently on hold. If the only test is for ignore case, then we can really ignore this.

> * `LoggingManagementControllerTestIT`: all 16 tests are being skipped. On top of the class, there's a comment regarding migrating to spring boot actuator after #1000, however, as far as I saw, actuator logs endpoint weren't enabled in #1000. Is there any ticket tracking this need (if that is a real need)? Also, if enabled, logs are not failing.

@fuss86 : Any thoughts on this?

> * `ProxyRepositoryConnectionPoolConfigurationServiceImplIntegrationTest`: Tests are disabled but all the 6 tests are passing.

Great, if these are working, you could re-enable them.

> * `MavenProxyRepositoryProviderTestIT`: One test being skipped stating that needs to Docker needs to be in place. Is the docker setup done?

Who put the comment? Does the test actually pass?

> * `ChecksumCacheManagerTest`: 2 tests disabled but are passing

Great, if these are working, you could re-enable them.

**Comment by fuss86** [comment]
> > * `LoggingManagementControllerTestIT`: all 16 tests are being skipped. On top of the class, there's a comment regarding migrating to spring boot actuator after #1000, however, as far as I saw, actuator logs endpoint weren't enabled in #1000. Is there any ticket tracking this need (if that is a real need)? Also, if enabled, logs are not failing.
> 
> @fuss86 : Any thoughts on this?

logging management is currently under reconstruction and migration to spring boot actuator (see also https://github.com/strongbox/strongbox/pull/1440). In my opinion we won't need `LoggingManagementController` soon. I talked with @steve-todorov about this so can also add a comment here. I suggest to drop `LoggingManagementController` and `LoggingManagementControllerTestIT` after successful migration.

**Comment by carlspring** [comment]
> * `ProxyRepositoryConnectionPoolConfigurationServiceImplIntegrationTest`: Tests are disabled but all the 6 tests are passing.

This test should probably be renamed to `ProxyRepositoryConnectionPoolConfigurationServiceImplTestIT`.

**Comment by ykabusalah** [comment]
Has this one been taken completely? Or is there any room for me to try it?

**Comment by PedroHReis22** [comment]
@carlspring, @fuss86 , thanks I'll apply the changes and create the PR.

**Comment by carlspring** [comment]
@ykabusalah ,

Yes, @PedroHReis22 is already looking into this. Would you mind looking around our issue tracker for some other task and joining our [chat channel](https://chat.carlspring.org/)? :)
